client/modules/core/main.js

```diff
@@ -387,25 +388,35 @@ export default {
   },
 
   getUserPreferences(packageName, preference, defaultValue) {
-    const user = Meteor.user();
-
-    if (user) {
-      const profile = Meteor.user().profile;
-      if (profile && profile.preferences && profile.preferences[packageName] && profile.preferences[packageName][preference]) {
-        return profile.preferences[packageName][preference];
+    getDep(`${packageName}.${preference}`).depend();
+    if (Meteor.user()) {
```

Comment:
- Is `Meteor.user()` still necessary?

Reply:
@prinzdezibel It's just an optimisation to get settings only when the user is actually logged in.
As we have removed all the dependencies on by not storing anything in `Meteor.user().profile.preferences` this shouldn't cause components to re-render.
Why did you think it would be a good idea to remove it? I might be missing something.

Reply:
@prinzdezibel It's just an optimisation to get settings only when the user is actually logged in.
As we have removed all the dependencies on by not storing anything in `Meteor.user().profile.preferences` this shouldn't cause components to re-render.
Why did you think it would be a good idea to remove it? I might be missing something.